docs/content/en/command-line-interface/cli.md

```diff
@@ -0,0 +1,137 @@
+---
+date: 2022-3-25
+title: Command Line Interface
+draft: false
+---
+
+<!--Copyright (c) Coherent Labs AD. All rights reserved. Licensed under the MIT License. See License.txt in the project root for license information. -->
+
+The coherent-guic-cli is a tool that helps you create and develop components from the GameUIComponents suite. It can also setup a development environment with live reload that will allow you to easily create a UI using the components.
+
+# Getting Started
+
+To install the cli open a console and type:
+
+```
+npm i -g coherent-guic-cli
+```
+
+This will install the cli as a global module and will make it available from everywhere.
+
+## Available Commands
+
+|Command   |Description   |Arguments   |Usage   |
+|---|---|---|---|
+|create-project               |Create a new project from template                 | `<template>` `<directory>`|`coherent-guic-cli create-project Webpack ./my-folder`|
+|create | [name], [directory] |           Create a new component | `coherent-guic-cli create my-component ./my-folder`|
+|build |[--watch]        | start the server|`coherent-guic-cli build`|
+|start:demo| N/A|              Start a development server and host the demo|`coherent-guic-cli start:demo`|
+|build:demo| [--dev] |      Create a production bundle of the demo.
+
+
+## Usage
+
+### Setting up an Environment

```

Comment:
This section is a bit strange. You are giving an example of how to create a project with webpack and then an example is given with vite. The commands are the same for both - `npm run dev`.
Either combine both into a single example and usage or split both sections with headings such as
```
#### Project with webpack

...

#### Project with Vite

...
```
for example.

Reply:
Added clarification for vite and webpack. The command is the same - yes, it was on purpose.

Reply:
Better! 🙇